persist-cli/src/main/java/io/ballerina/persist/nodegenerator/SyntaxTreeGenerator.java

```diff
@@ -134,8 +181,8 @@ public static HashMap<String, String> readToml(Path configPath, String name) thr
                 }
             }
             if (!persistConfigs) {
-                throw new BalException("Persist client related config doesn't exist in Config.toml.\n" +
-                        "You should add database configurations under <org name>.<pkg name>.clients ");
+                throw new BalException("Persist client related config doesn't exist in Persist.toml.\n" +
+                        "You should add database configurations or placeholders under [database] ");
```

Comment:
```suggestion
                throw new BalException("Persist client related config doesn't exist in Persist.toml.\n" +
                        "You should add [database] table with configurations values or placeholders under [database] ");
```

Reply:
Shall we remove the under [database] path at the end. It might sound confusing as it's repeated?

Reply:
+1. Amended the suggestion